### PR TITLE
Feature/#278 first bad version

### PR DESCRIPTION
**Pull request by Rwatana**
Please check my code

### CODE REVIEW

**Comment by ryujumoto** [comment]
There is no files.

**Comment by Rwatana** [comment]
I checked that the file exists.
I 'm sorry.

File: 278_first_bad_version.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class Solution:
+    def firstBadVersion(self, n: int) -> int:
+        up, down = 1, n
+        middle = 0
+        while up < down:
+            middle = (up + down) // 2
+            if isBadVersion(middle) is True:
```
Comment (ryujumoto) [inline 278_first_bad_version.py]: if isBadVersion(middle) is True: -> if isBadVersion(middle):
Don't use "==" or "is" to compare boolean values with True or False (see PEP8).

File: 278_first_bad_version.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class Solution:
+    def firstBadVersion(self, n: int) -> int:
+        up, down = 1, n
+        middle = 0
```
Comment (ryujumoto) [inline 278_first_bad_version.py]: Not necessary.

**Review by ryujumoto — COMMENTED** [review]
Done.

**Comment by Rwatana** [comment]
I think I checked all points. Please check my code.

**Review by ryujumoto — APPROVED** [review]

**Pull request merged**